### PR TITLE
Language tweaks

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: aldryn_jobs/models.py [hunk 1]
```diff
@@ -136,18 +136,18 @@ def default_jobs_attachment_upload_to(instance, filename):
...
+                        'will change. Clear it to have the slug re-created.'))
     )
 
     supervisors = models.ManyToManyField(
         get_user_model_for_fields(), verbose_name=_('supervisors'),
         # FIXME: This is mis-named should be "job_categories"?
         related_name='job_opening_categories',
-        help_text=_('Those people will be notified via e-mail when new '
-                    'application arrives.'),
+        help_text=_('Supervisors will be notified via e-mail when a new '
```
Comment (evildmp) [inline aldryn_jobs/models.py]: Can we standardise on "email" please?

**Comment by evildmp** [comment]
Looks good, made a small note about e-mail/email.

**Comment by intellectronica** [comment]
:+1: and i agree with @evildmp about `email`.

File: aldryn_jobs/models.py (hunk 1 above)
Comment (mkoistinen) [inline aldryn_jobs/models.py]: :+1:

**Pull request merged**